api/views/voto_serializer.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import serializers, generics
+from api.model.voto import Voto
+
+
+class VotoSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Voto
+        fields = ("id_votacao", "id_parlamentar_parlametria", "voto")
+
+
+class VotosByParlamentar(generics.ListAPIView):
+    '''
+    Retorna todas as votaões
+    '''
+
+    serializer_class = VotoSerializer
+
+    def get_queryset(self):
+        '''
+        Retorna as votações de uma casa
+        '''
+
+        id_parlamentar = self.kwargs["id_parlamentar"]
+
+        votos = Voto.objects().all()
```

Comment:
```suggestion
        votos = Voto.objects.all()
```
Com os parênteses aqui deu o erro:
`TypeError: 'Manager' object is not callable`